index.css

```diff
@@ -0,0 +1,91 @@
+.product-category {
+  width: 80%;
+  margin: 5rem auto;
+}
+
+.product-category .category-heading {
+  padding: 0.8rem;
+}
+
+.category-display {
+  margin: 3rem 0;
+  display: grid;
+  grid-gap: 1rem;
+
+  grid-template-columns: repeat(auto-fit, minmax(18rem, 1fr));
+}
+
+.card-with-text-overlay {
+  opacity: 1;
+}
+.card {
+  margin: 0;
+  width: 18rem;
+}
+
+.text-overlay {
+  top: 8rem;
+  background-color: rgba(253, 231, 231, 0.65);
+  bottom: 9rem;
+}
+
+.card-with-text-overlay img {
+  height: 20rem;
+  border: none;
+  border-radius: none;
+}
+
+.banner-section {
+  height: 50rem;
+  position: relative;
+}
+
+.banner-section img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.banner-section .banner-text-overlay {
+  width: 100%;
+  height: 100%;
+  background-color: rgba(54, 54, 54, 0.65);
+  z-index: 1;
+  position: absolute;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: var(--lynx-white-color);
+  font-weight: var(--bold-font);
+}
+
+.banner-section .banner-text-overlay h2 {
+  margin-bottom: 1rem;
+}
+
+.banner-section .banner-text-overlay button {
```

Comment:
Just a suggestion , this will apply the style to every button in ```.banner-section```

Reply:
Actually, there is only one button in the banner section right now, that is why I have written like this. In the future, if I add more buttons then I will change them accordingly.